test/test_runner.c

```diff
@@ -6,6 +6,8 @@ int main(int argc, char **argv) {
    "startVentilation_afterOneInspiration_openInspiratoryValve");
   test(startVentilation_duringInspiration_doesNotopenInspiratoryValve, 
   "startVentilation_duringInspiration_doesNotopenInspiratoryValve");
+  test(calculateBreathCycleDuration_15BPM, 
```

Comment:
unit test name should be of the form:

functionUnderTest_precondition_expectedResult

I suggest something like :
```suggestion
  test(calculateBreathCycleDuration_defaultRR_returns40000), 
```